pxtlib/emitter/assembler.ts

```diff
@@ -390,17 +387,14 @@ namespace ts.pxtc.assembler {
                 if (this.finalEmit)
                     this.directiveError(lf("unknown label: {0}", name));
                 else
+                    // Is this a joke?
```

Comment:
There needs to be a value here, and in the first emit labels might not be resolved yet. I think there was some reason not to use 0